### ISSUE TITLE
When downloading fullTextXML for a particular pmcid, if eupmc returns a 404, getpapers crashes 

### ISSUE THREAD

**Issue by anusharanganathan** [ContentMine/getpapers#52]
The url [http://www.ebi.ac.uk/europepmc/webservices/rest/PMC1693416/fullTextXML](http://www.ebi.ac.uk/europepmc/webservices/rest/PMC1693416/fullTextXML) returns a 404 and this crashes getpapers.

Command executed    
```
$ getpapers -q 'dinosaurs' -l 'debug' -x -s -p -a -o dinosaursOutput4 >> dinosaursOutput4.log
```

The output at the location it crashed 

```
debug: Downloading XML: http://www.ebi.ac.uk/europepmc/webservices/rest/PMC1693416/fullTextXML
Downloading files [------------------------------] 0% (eta 0.0s)
/home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/lib/eupmc.js:333
          fourohfour();
          ^
TypeError: undefined is not a function
    at /home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/lib/eupmc.js:333:11
    at /home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/node_modules/got/index.js:152:6
    at BufferStream.<anonymous> (/home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/node_modules/got/node_modules/read-all-stream/index.js:52:3)
    at BufferStream.EventEmitter.emit (events.js:117:20)
    at finishMaybe (/home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/node_modules/got/node_modules/read-all-stream/node_modules/readable-stream/lib/_stream_writable.js:502:14)
    at endWritable (/home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/node_modules/got/node_modules/read-all-stream/node_modules/readable-stream/lib/_stream_writable.js:512:3)
    at BufferStream.Writable.end (/home/workshop/.nvm/v0.10.24/lib/node_modules/getpapers/node_modules/got/node_modules/read-all-stream/node_modules/readable-stream/lib/_stream_writable.js:477:5)
    at IncomingMessage.onend (_stream_readable.js:483:10)
    at IncomingMessage.g (events.js:180:16)
    at IncomingMessage.EventEmitter.emit (events.js:117:20)
```

Version of getpapers: 0.3.0

**Comment by anusharanganathan** [comment]
This could be reproduced easily with the following command
```
getpapers --query  'PMCID:PMC1693416' -l 'debug' -x -s -p -a -o Output
```

**Comment by Blahah** [comment]
fixed in `0.4.1`
